README.md

```diff
@@ -30,6 +30,6 @@
 - Use `_` between a job name and the [job template variables](https://docs.openstack.org/infra/jenkins-job-builder/definition.html#job-template)
 - Use standard capitalization rules, template variables can be an exception to this
 - Examples:
-  - `RPC-AIO_{series}-{context}-{ztrigger}`
-  - `RPC-AIO_master-swift-periodic`
-  - `Merge-Trigger-JJB`
\ No newline at end of file
+  - `RPC-AIO_{series}-{image}-{scenario}-{action}-{ztrigger}`
+  - `RPC-AIO_master-xenial-swift-deploy-periodic`
```

Comment:
The more complete approach would have for example:
- `RPC-AIO_master-xenial-swift-deploy-from_nothing-periodic`
- `RPC-AIO_master-trusty-swift-upgrade-from_14.1.0-periodic`
- `RPC-AIO_newton14.1-trusty-swift-leapfrog-from_kilo-periodic`
- `RPC-AIO_newton14.1-trusty-swift-leapfrog-from_liberty-periodic`

Adding an axis for the ``upgrade from`` version makes sense to me.